src/main/scala/org/broadinstitute/hail/sparkextras/OrderedRDD.scala

```diff
@@ -87,31 +87,33 @@ object OrderedRDD {
 
     val sortedKeyInfo = keyInfo.sortBy(_.min)
     val partitionsSorted = sortedKeyInfo.zip(sortedKeyInfo.tail).forall { case (p, pnext) =>
-      val r = p.max < pnext.min
+      val r = p.max <= pnext.min
```

Comment:
Should we relax this?  Once you have to stream through the previous partition, it doesn't really matter if you take a little or a lot.